src/ast.rs

```diff
@@ -23,17 +23,17 @@ pub enum LhsTransformations {
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum BinaryOperator {
-    Equals,         // ==
-    NotEquals,      // !=
-    Regex,          // ~
-    Prefix,         // ^=
-    Postfix,        // =^
-    Greater,        // >
-    GreaterOrEqual, // >=
-    Lesser,         // <
-    LesserOrEqual,  // <=
-    In,             // in
-    NotIn,          // not in
+    Equals,
```

Comment:
Why did we remove these comments? I think these comments make the code more readable.

Reply:
> Why did we remove these comments? I think these comments make the code more readable.

I think these enum items are clear enough

Reply:
> We should keep the comment, some reader might not be familiar with what the operator looks like immediately.

We already have this, which I think is more of a sound than a comment
```
impl fmt::Display for BinaryOperator {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        use BinaryOperator::*;

        write!(
            f,
            "{}",
            match self {
                Equals => "==",
                NotEquals => "!=",
                Regex => "~",
                Prefix => "^=",
                Postfix => "=^",
                Greater => ">",
                GreaterOrEqual => ">=",
                Less => "<",
                LessOrEqual => "<=",
                In => "in",
                NotIn => "not in",
                Contains => "contains"
            }
        )
    }
}
```

Reply:
Comments can let the readers understand the code clearly, removing them will not make any benefit.